### PR TITLE
update publisher video tile kebab menu

### LINKED ISSUE #2028: Update the context menu items of publisher video tile
https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-Components?node-id=3630:116415

AC to be defined

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #2028 

**Design**
https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-components?node-id=3630%3A116415

### CODE REVIEW

**Review by drillprop — APPROVED** [review]
Looks good to me!

File: packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx [hunk 1]
```diff
@@ -122,8 +123,8 @@ export const VideoForm: React.FC<VideoFormProps> = React.memo(({ onSubmit, setFo
     }
     setCachedEditedVideoId(editedVideoInfo.id)
 
-    reset(tabData)
-  }, [tabData, tabDataLoading, reset, editedVideoInfo.id, cachedEditedVideoId, setValue])
+    reset({ ...tabData, mintNft: tabData.mintNft || !!mintNft })
```
Comment (kdembler) [inline packages/atlas/src/views/studio/VideoWorkspace/VideoForm/VideoForm.tsx]: Not sure about this, `mintNft` will be set as a form default value. We have the cancel button and if you click it, all edits should be reset. However, `mintNft` will not have the proper value set. You could leave the `reset` as it is and add `setValue` after it to override. You probably will need to put `setValue` inside a `setTimeout`, otherwise it doesn't work

**Comment by rafalpawlow** [comment]
@kdembler according to our rocket chat conversation, I made the scroll animation according to the user story
[https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=2365:87248](https://www.figma.com/file/xm6vBdSnQDroc57rGfGtPM/Video-workspace?node-id=2365%3A87248)

File: packages/atlas/src/components/_video/VideoTilePublisher/VideoTilePublisher.tsx [hunk 2]
```diff
@@ -35,21 +38,24 @@ import { VideoTile } from '../VideoTile'
...
 export const DELAYED_FADE_CLASSNAME = 'delayed-fade'
 
 export const VideoTilePublisher: React.FC<VideoTilePublisherProps> = React.memo(
-  ({ id, onEditClick, onDeleteVideoClick, onReuploadVideoClick }) => {
+  ({ id, onEditClick, onDeleteVideoClick, onReuploadVideoClick, onMintNftClick }) => {
     const { copyToClipboard } = useClipboard()
     const { isLoadingThumbnail, thumbnailPhotoUrl, loading, video, videoHref } = useVideoTileSharedLogic({
       id,
     })
+    const { nft } = useNft(video?.id || '')
```
Comment (kdembler) [inline packages/atlas/src/components/_video/VideoTilePublisher/VideoTilePublisher.tsx]: The NFT should be extracted from `video` object, this will cause `GetNft` query to be run for every separate tile rendered

File: packages/atlas/src/views/studio/MyVideosView/MyVideosView.tsx [hunk 3]
```diff
@@ -80,6 +80,7 @@ export const MyVideosView = () => {
       : _drafts.slice().sort((a, b) => new Date(a.updatedAt).getTime() - new Date(b.updatedAt).getTime())),
   ]
 
+  useChannelNfts(activeChannelId)
```
Comment (kdembler) [inline packages/atlas/src/views/studio/MyVideosView/MyVideosView.tsx]: Why is this here?

**Comment by kdembler** [comment]
Loading state for video tiles is broken:

https://user-images.githubusercontent.com/12646744/160563966-b68bafbb-f044-4d71-92d0-f2222a593a66.mov

File: packages/atlas/src/views/studio/MyVideosView/MyVideosView.tsx (hunk 3 above)
Comment (rafalpawlow) [reply packages/atlas/src/views/studio/MyVideosView/MyVideosView.tsx]: I did this to cache all publisher's nfts. Is it a bad idea? 😅

Comment (kdembler) [reply packages/atlas/src/views/studio/MyVideosView/MyVideosView.tsx]: The NFTs should be fetched as part of the query that fetches videos already, no?

Comment (rafalpawlow) [reply packages/atlas/src/views/studio/MyVideosView/MyVideosView.tsx]: Yeah but we need to fetch it one more time in `useGetNftSlot`

**Review by kdembler — APPROVED** [review]
